airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/SqlGenerator.java

```diff
@@ -4,10 +4,16 @@
 
 package io.airbyte.integrations.base.destination.typing_deduping;
 
+
+import java.util.List;
 import java.util.Optional;
+import java.util.Set;
 
 public interface SqlGenerator<DialectTableDefinition> {
 
+  Set<String> FINAL_TABLE_AIRBYTE_COLUMNS = Set.of("_airbyte_raw_id", "_airbyte_extracted_at", "_airbyte_meta");
+  String SOFT_RESET_SUFFIX = "_ab_soft_reset";
```

Comment:
nit: Maybe these should be static constants somewhere

Reply:
IDE yelled at me: all interface members are "public, static, final" by default so static keyword is redundant